draft-ietf-quic-transport.md

```diff
@@ -5830,17 +5830,18 @@ IANA \[SHALL add/has added] a registry for "QUIC Transport Parameters" under a
 "QUIC Protocol" heading.
 
 The "QUIC Transport Parameters" registry governs a 16-bit space.  This space is
-split into two spaces that are governed by different policies.  Values with the
-first byte in the range 0x00 to 0xfe (in hexadecimal) are assigned via the
-Specification Required policy {{!RFC8126}}.  Values with the first byte 0xff are
-reserved for Private Use {{!RFC8126}}.
+split into three spaces that are governed by different policies.  Values between
+0x0000 and 0x003f (in hexadecimal) are assigned via the Standards Action or IESG
```

Comment:
I don't understand these policies well enough -- why do we need a separate space for Standards Action? Isn't that covered under Specification Required?

Reply:
@janaiyengar `Standards Action` means WG-track RFC so it requires some amount of IETF consensus, whereas `Specification Required` just means someone wrote a draft. See RFC 8126 for details

Reply:
Not even an I-D on the Datatracker.  If you can point to a URL of a document somewhere that describes it, that's sufficient for Specification Required.